The job **`x86_64-fuchsia`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/11563225524/job/32186087017) [(plain)](https://github.com/rust-lang-ci/rust/commit/6ccb9feea3997358af7e335e36b4060608216fcb/checks/32186087017/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
file:.git/config remote.origin.url=https://github.com/rust-lang-ci/rust
file:.git/config remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
file:.git/config gc.auto=0
file:.git/config http.https://github.com/.extraheader=AUTHORIZATION: basic ***
file:.git/config branch.try.remote=origin
file:.git/config branch.try.merge=refs/heads/try
file:.git/config remote.upstream.fetch=+refs/heads/*:refs/remotes/upstream/*
file:.git/config submodule.library/backtrace.active=true
file:.git/config submodule.library/backtrace.url=https://github.com/rust-lang/backtrace-rs.git
file:.git/config submodule.library/stdarch.active=true
---
#                                                                             #
###############################################################################

+ /checkout/obj/fuchsia/scripts/../.jiri_root/bin/fx set --args 'rustc_prefix = "/checkout/obj/install"' --args 'rustc_version_string = "e149462ae390d84382838319452a244168b9617f  -"' --args 'rust_cap_lints = "warn"' --args 'rustc_use_response_files = false' --args 'rust_one_rlib_per_dir = false' --args 'restat_rust = false' --args 'verify_depfile = false' --args 'debuginfo = "none"' --args 'disable_elf_checks = true' --with //bundles/buildbot/minimal workbench_eng.x64
WARNING: Please opt in or out of fx metrics collection.
You will receive this warning until an option is selected.
To check what data we collect, run `fx metrics`
To opt in or out, run `fx metrics <enable|disable>

ERROR at //build/assembly/assembly_input_bundle.gni:668:36: Empty directory path.
              source = rebase_path("${entry.source}", root_build_dir)
You can't use empty strings as directories.
You can't use empty strings as directories.
See //bundles/assembly/BUILD.gn:2542:1: whence it was called.
assembly_input_bundle("fshost_common") {
^---------------------------------------
See //BUILD.gn:1580:5: which caused the file to be included.
    "//bundles/assembly",
    ^-------------------
ERROR: error running gn gen: exit status 1
  network time: Mon, 28 Oct 2024 22:26:18 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/src/ci/docker/host-x86_64/x86_64-fuchsia/build-fuchsia.sh
+++ b/src/ci/docker/host-x86_64/x86_64-fuchsia/build-fuchsia.sh
@@ -35,7 +35,7 @@ PICK_REFS=()
 # commit hash of fuchsia.git and some other repos in the "monorepo" checkout, in
 # addition to versions of prebuilts. It should be bumped regularly by the
 # Fuchsia team – we aim for every 1-2 months.
-INTEGRATION_SHA=1c5b42266fbfefb2337c6b2f0030a91bde15f9e9
+INTEGRATION_SHA=9f632bb7446d5a6af2998f1a0ebdb4b8ea2f4511
 
 checkout=fuchsia
 jiri=.jiri_root/bin/jiri

